cmd/params/params.go

```diff
@@ -599,9 +602,18 @@ func LoadOfflineParams(v *viper.Viper) (Offline, error) {
 		return Offline{}, errors.New("argument --sync-offline-activity must be \"none\" or a positive integer number")
 	}
 
+	switch {
+	case !v.IsSet("print-offline-heartbeats"):
```

Comment:
Do we need this? The `--print-offline-heartbeats` param requires an integer afterwards so it shouldn't ever need to use the default.

Reply:
No we don't. I also have that fix in mind for `sync-offline-activity` as well.

Reply:
Fixed it and merged.